crates/flux-fhir-analysis/src/lib.rs

```diff
@@ -86,6 +88,10 @@ fn func_sort(genv: GlobalEnv, def_id: FluxId<MaybeExternId>) -> rty::PolyFuncSor
     }
 }
 
+fn func_span(genv: GlobalEnv, def_id: FluxId<MaybeExternId>) -> Span {
+    genv.fhir_spec_func_body(def_id.local_id()).unwrap().span
+}
```

Comment:
We can make this generic for any flux definition, not just functions. Let's call it `flux_def_ident_span`. We must be careful because a `FluxId` can point either to a `FluxItem` or to an `AssocRefinement`. Also we probably want the span of the identifier not of the entire definition.